README.md

```diff
@@ -1,23 +1,29 @@
-# GDX Looker Google Block
-This is for the Government of British Columbia’s instance their Google API LookerML project. LookerML is a language for describing dimensions, aggregates, calculations and data relationships in a SQL database
+# GDX Analytics Looker Google Block
+This is the Looker project for the Government of British Columbia’s instance their Google API.
 
-# Features
+## Features
```

Comment:
Rephrase to describe the features in this repo

Reply:
Done